test/com/google/cose/Sign1MessageTest.java

```diff
@@ -32,10 +34,12 @@
 public class Sign1MessageTest {
   @Test
   public void testDeserialize() throws CborException, CoseException {
-    Sign1Message message = Sign1Message.deserialize(TestUtilities.hexStringToByteArray("8441A0A20126044"
-      + "2313154546869732069732074686520636F6E74656E742E584087DB0D2E5571843B78AC33ECB2830DF7B6E0A4"
-      + "D5B7376DE336B23C591C90C425317E56127FBE04370097CE347087B233BF722B64072BEB4486BDA4031D27244F"
-    ));
+    Sign1Message message =
+        Sign1Message.deserialize(
+            TestUtilities.hexStringToByteArray(
+                "8441A0A20126044"
+                    + "2313154546869732069732074686520636F6E74656E742E584087DB0D2E5571843B78AC33ECB2830DF7B6E0A4"
```

Comment:
This will cross the 100 character per line limit. See if we can maintain the previous style.

Reply:
Done, although, as a comment, the original style also exceeded 100 characters. Not sure if that should also be addressed.